active-rfcs/0000-style-unification.md

```diff
@@ -0,0 +1,91 @@
+- Start Date: 2021-12-14
+- Reference Issues: https://github.com/withastro/rfcs/discussions/28
+- Implementation PR: 
+
+# Summary
+
+Clean up and finalize how CSS is used in an Astro project.
```

Comment:
It would be nice if the summary included a quick list of things being added or removed. I think

Reply:
Added another line here to summarize the new method being proposed.